### DISCUSSION TITLE
Free space calculation

### DISCUSSION THREAD

**Discussion by cyberndj** [trapexit/mergerfs#1261] [Q&A]
Howdy,
Hopefully my search-foo is failing, but on free space calculations (ex, mfs), is the free space calculations based on the volume of free space (Ex # of GB's) or percent?

If its by #ofGB's, looks like mergerfs.balance (and other mergerfs tools) is the right tool to use to make balanced on percent.
Thanks!

**Comment by trapexit** [comment]
It is just the size. As said in the policy descriptions.
